### ISSUE TITLE
tomcat_permgem_memory is not defaulted

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/ansible-hydra#105]
```
TASK: [hydra-stack/install | create tomcat config and java options] *********** 
fatal: [default] => {'msg': "AnsibleUndefinedVariable: One or more undefined variables: 'tomcat_permgem_memory' is undefined", 'failed': True}
fatal: [default] => {'msg': "AnsibleUndefinedVariable: One or more undefined variables: 'tomcat_permgem_memory' is undefined", 'failed': True}

FATAL: all hosts have already failed -- aborting
```

**Comment by acozine** [comment]
That's weird. I see a default: 
https://github.com/curationexperts/ansible-hydra/blob/master/roles/hydra-stack/install/defaults/main.yml#L8
Did I make a typo somewhere?

**Comment by mark-dce** [comment]
@acozine yes - I figured it out and fixed it - see 1704b4031a2fa3869a8754ab0a6ca18547a60f7c
"permgem" vs. "permgen" - i.e. you had typed m vs. n
